deploy/konnector/90-deployment.yaml

```diff
@@ -18,14 +18,18 @@ spec:
       restartPolicy: Always
       serviceAccountName: konnector
       containers:
-      - name: konnector
-        image: IMAGE
-        env:
-        - name: POD_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
-        - name: POD_NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
\ No newline at end of file
+        - name: konnector
```

Comment:
*Nitpick*: this block's indentation was changed. IMHO it's better to avoid such changes on a fork, since it will lead to merge conflicts whenever the file is modified upstream. Did you change it on purpose?